specification/compute/resource-manager/Microsoft.Compute/GalleryRP/stable/2023-09-01/gallery.json

```diff
@@ -3144,6 +3333,200 @@
+        "id",
+        "location",
+        "name",
+        "properties",
+        "type"
+      ]
+    },
+    "GalleryServiceArtifactTargetLocation": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string"
+        },
+        "vmArtifactsProfiles": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/VMArtifactsProfile"
+          }
+        }
+      },
+      "required": [
+        "name"
+      ]
+    },
+    "VMArtifactsProfile": {
+      "type": "object",
+      "properties": {
+        "artifactProfiles": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ArtifactProfile"
+          }
+        },
+        "description": {
+          "type": "string"
+        },
+        "name": {
+          "type": "string"
+        },
+        "upgradeProviderInfo": {
+          "$ref": "#/definitions/UpgradeProviderInfo"
+        },
+        "upgradeSdpPolicy": {
+          "$ref": "#/definitions/UpgradeSDPPolicy"
+        }
+      },
+      "required": [
+        "artifactProfiles",
+        "name",
+        "upgradeProviderInfo",
+        "upgradeSdpPolicy"
+      ]
+    },
+    "ArtifactProfile": {
+      "type": "object",
+      "properties": {
+        "description": {
+          "type": "string"
+        },
+        "imageReference": {
+          "$ref": "#/definitions/ImageReference"
+        }
+      }
+    },
+    "ImageReference": {
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string"
+        },
+        "initialVersion": {
+          "type": "string"
+        },
+        "publisher": {
+          "type": "string"
+        },
+        "offer": {
+          "type": "string"
+        },
+        "sku": {
+          "type": "string"
+        }
+      },
+      "required": [
+        "initialVersion"
+      ]
+    },
+    "UpgradeProviderInfo": {
+      "type": "object",
+      "properties": {
+        "enableAutoTrigger": {
+          "type": "boolean"
+        },
+        "upgradeProviderParameters": {
+          "$ref": "#/definitions/UpgradeProviderParameters"
+        }
+      }
+    },
+    "UpgradeProviderParameters": {
+      "type": "object",
+      "properties": {
+        "ServiceGroup": {
+          "type": "string"
+        },
+        "ServiceIdentifier": {
+          "type": "string"
+        }
+      }
+    },
+    "UpgradeSDPPolicy": {
+      "type": "object",
+      "properties": {
+        "schedulingPolicy": {
+          "$ref": "#/definitions/SchedulingPolicy"
+        },
+        "type": {
+          "type": "string"
+        }
+      }
+    },
+    "SchedulingPolicy": {
+      "type": "object",
+      "properties": {
+        "disableRegularUpgrades": {
+          "type": "boolean"
+        },
+        "maxConcurrentResourceCountPerRegion": {
+          "type": "integer"
+        },
+        "upgradeWindow": {
+          "type": "string"
+        }
+      }
     }
```

Comment:
Do we need to have "patch" for vmprofile version updates? or is "put" suffice?
i.e.
```
"patch": {
        "tags": [
          "GalleryImageVersions"
        ],
        "operationId": "GalleryImageVersions_Update",